docs/api.rst

```diff
@@ -0,0 +1,45 @@
+API
+===
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
I wonder if you should do `3` here? Then the side bar will have:

```
# API
 ## Tracing
   ### ddtrace.patch_all
   ### ddtrace.patch
   ### ddtrace.Tracer
   ### ddtrace.Span
```

what do you think?

Reply:
Removed this as this directive didn't seem to be doing what I expected. Will fix in follow-up.

Reply:
I did create the section tree that you suggested.